scrapy/commands/crawl.py

```diff
@@ -54,8 +54,14 @@ def run(self, args, opts):
             raise UsageError("running 'scrapy crawl' with more than one spider is no longer supported")
         spname = args[0]
 
-        self.crawler_process.crawl(spname, **opts.spargs)
-        self.crawler_process.start()
-
-        if self.crawler_process.bootstrap_failed:
+        res = self.crawler_process.crawl(spname, **opts.spargs)
```

Comment:
:lipstick: Can we rename `res` to something that cannot be confused with `resource`, or mean `result` leading `res.result` to mean ‘result of result’? 

I’m thinking something in the lines of `process_defer`, `crawl_defer`… I’m not really sure, but `res` feels too ambiguous.

Reply:
good suggestion, I'll modify res to crawl_defer